app/src/main/java/com/example/coopertest/ResultActivity.kt

```diff
@@ -1,12 +1,110 @@
 package com.example.coopertest
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.Gravity
+import android.view.MenuItem
+import java.util.ArrayList
+import android.widget.TableLayout
+import android.widget.TableRow
+import android.widget.TextView
+import androidx.core.view.children
+import androidx.preference.PreferenceManager
+
 
 class ResultActivity : AppCompatActivity() {
 
+    private val objectContext : App = App(this)
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_result)
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
+        supportActionBar?.setTitle("Results")
+        val listOfResults= Storage().loadResults(this)
+        fillTable(listOfResults)
+    }
+
+    fun fillTable(listOfResults : ArrayList<Results>?){
+        val table = findViewById<TableLayout>(R.id.tableResults)
```

Comment:
Is it necessary to do find by Id in Kotlin? I thought just tableResults works ;)